dojo/settings/settings.dist.py

```diff
@@ -1173,6 +1173,7 @@ def saml2_attrib_map_format(dict):
     'Anchore Enterprise Policy Check': ['title', 'severity', 'component_name', 'file_path'],
     'Anchore Grype': ['title', 'severity', 'component_name', 'component_version'],
     'Aqua Scan': ['severity', 'vulnerability_ids', 'component_name', 'component_version'],
+    'AWS Prowler V3': ['severity', 'component_name', 'unique_id_from_tool'],
```

Comment:
Adding `unique_id_from_tool` to the hash code fields will remove the possibility of having deduplication across tools. Please either remove it or replace it with another field
```suggestion
    'AWS Prowler V3': ['severity', 'component_name'],
```